modules/calib3d/include/opencv2/calib3d.hpp

```diff
@@ -3773,7 +3773,8 @@ namespace fisheye
         CALIB_FIX_K4                = 1 << 7,
         CALIB_FIX_INTRINSIC         = 1 << 8,
         CALIB_FIX_PRINCIPAL_POINT   = 1 << 9,
-        CALIB_ZERO_DISPARITY        = 1 << 10
+        CALIB_ZERO_DISPARITY        = 1 << 10,
+        CALIB_FIX_FOCAL_LENGTH      = 1 << 11
```

Comment:
Please add/extend bindings tests for constants.
There is some mess with constants with the same name and different values.

Reply:
@alalek It's already on place: https://github.com/opencv/opencv/blob/master/modules/calib3d/misc/java/test/Calib3dTest.java#L634.